### PR TITLE
[Hub Generated] Review request for Microsoft.Capacity to add version preview/2019-04-01

### PR DESCRIPTION
**Pull request by gautam714**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/gautam714/azure-rest-api-specs/dev-reservations-Microsoft.Capacity-2019-04-01-renewProperties...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json [hunk 1]
```diff
@@ -862,15 +862,27 @@
         "purchaseProperties": {
           "$ref": "#/definitions/PurchaseRequest"
         },
-        "lockedPriceTotal": {
+        "pricingCurrencyTotal": {
```
Comment (ryansbenson) [inline specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json]: This is categorized as a breaking change and would require a new api version
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Comment (gautam714) [reply specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json]: Old property ‘lockedPriceTotal’ and new properties ‘billingCurrencyTotal’ and ‘pricingCurrencyTotal’ are part of the same work which is not released to Prod yet.
Nobody has taken a dependency on these properties as of yet. The change in properties has been made after an internal review.
So, this is not a breaking change.

**Review by ryansbenson — APPROVED** [review]
Approved from ARMs side

File: specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json (hunk 1 above)
Comment (ryansbenson) [reply specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json]: This is a documentation issue as the properties being changed haven't gone into the runtime of the service. This is also part of a read-only response property so it's non breaking anyways.

**Review by erich-wang — APPROVED** [review]

**Pull request merged**